hlevels.tex

```diff
@@ -2109,6 +2109,19 @@ \section{Modalities}
   Show that if $f : A \to B$ is $n$-connected, then $\trunc kf : \trunc kA \to \trunc kB$ is also $n$-connected.
 \end{ex}
 
+\begin{ex}\label{ex:categorical-connectivity}
+  We say a type $A$ is \define{categorically connected}\indexdef{connected!categorically} if for every types $B, C$ the canonical map $e_{A, B, C}:(A\to B) + (A\to C) \to A \to B + C$ defined by
```

Comment:
One more suggestion in the exercise statement: while this is technically correct as written, I think it would be clearer with some extra parentheses, to emphasize that `e_{A,B,C}` is a function from a coproduct of function types to a function type, e.g.

```
e_{A,B,C} : ((A\to B) + (A\to C)) \to (A\to (B + C))
```